core/rewards/engine.go

```diff
@@ -395,7 +395,7 @@ func (e *Engine) distributePayout(ctx context.Context, po *payout) {
 				Asset:  po.asset,
 				Amount: amt.Clone(),
 			},
-			Type:      types.TransferTypeRewardPayout,
+			Type:      proto.TransferType(po.rewardType),
```

Comment:
That doesn't seem right.

Reply:
I did it because of this comment - https://github.com/vegaprotocol/vega/issues/7012#issuecomment-1334978981
I interpreted it wrong then. Should I only leave the transfer type naming change ?

Reply:
All right, reverted back this change; only the renaming stays according to what Zohar suggested

Reply:
seems good now.